docs/tutorial.md

```diff
@@ -0,0 +1,91 @@
+# Tutorial
+
+After the operator is installed create an instance of 
+`AWSLoadBalancerController`. Note that since only a single instance of the 
+aws-load-balancer-controller can be installed in a cluster only the resource 
+with the name `cluster` will be reconciled by the operator.
+
+## AWSLoadBalancerController resource
+
+```yaml
+apiVersion: networking.olm.openshift.io/v1alpha1
+kind: AWSLoadBalancerController
+metadata:
+  name: cluster
+spec:
+  subnetTagging: Auto
+  additionalResourceTags:
+    example.org/cost-center: 5113232
+    example.org/security-scope: staging
+  ingressClass: cloud
+  config:
+    replicas: 2
+  enabledAddons:
+    - AWSShield
+    - AWSWAFv2
+```
+
+The spec of `AWSLoadBalancerController` resource has fields which are used 
+to configure the instance of `aws-load-balancer-controller`.
+
+### subnetTagging
+
+This field can take two values:
+* Auto
+* Manual
+
+When the value is set to `Auto` the operator attempts to determine the 
+subnets which belong to the cluster and tag them appropriately. It uses the 
+following logic.
+
+1. Fetch all the subnets with the tag key `kubernetes.io/cluster/$CLUSTER_ID`.
+2. If the subnet has the tag `kubernetes.io/role/internal-elb` then it's an 
+   internal subnet.
+3. Any subnets without the internal subnet tag are automatically classified 
+   as public subnets.
+4. The tag `kubernetes.io/role/elb` is added to the public subnets.
+
+__Note:__ The operator cannot determine the role correctly if the internal 
+subnet tags are not present on internal subnet. So if your cluster is 
+installed on User-Provisioned Infrastructure then you should manually tag 
+the subnets with the appropriate role tags and set the subnet tagging policy 
+to `Manual`
+
+### additionalResourceTags
+
+The tags are used by the controller when it provisions AWS resources. These 
+tags are added to the resource in addition to the cluster tag.
```

Comment:
These tags will be used by the controller when it provisions AWS resources. They are added to the resource in addition to the cluster tag.